cognite/extractorutils/logging_prometheus.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from prometheus_client import Counter
+
+log_entries = Counter(
+    "cognite_python_logging_messages_total", "Count of log entries by logger and level.", ["logger", "level"]
```

Comment:
@pavelzubarev should we change the name to `cognite_python_logging_messages`, so don't end up with `_total_total` in the name.

Reply:
good catch!